src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx

```diff
@@ -23,150 +23,144 @@ import { FiatValue } from 'components/CurrencyInputPanel/FiatValue'
 import { WithClassName } from 'types'
 import { formatSmart } from 'utils/format'
 import { SHORT_PRECISION } from 'constants/index'
-import { FeeInformationTooltipWrapper } from 'components/swap/FeeInformationTooltip'
+import {
```

Comment:
Basically moving all of the styled component to the `index` file, and import it here in the Mod file.

Reply:
but why? if uni makes changes we want to adapt in say `@src/components/CurrencyInputPanel` i would check against `@custom/components/CurrencyInputPanel/CurrencyInputPanelMod` to see the differences but now I have to check also against the index file. 

In this case it's better to keep the former and leave them there. If we _create new_ components, then we should add them to the index

Reply:
Makes sense, in that case I'll have to move them back. 

I was more thinking, if we update to the latest UNI code, we diff the Mod file (albeit commented out stuff) vs. their new code. But either works for me.

Reply:
yes please move them back 🙇

Reply:
Moved back.